lib/pdf_fill/extras_generator_v2.rb

```diff
@@ -30,7 +30,32 @@ def sorted_subquestions
         end
       end
 
+      def checklist_group?
```

Comment:
Because the checklist group is structurally the same as a regular question with subquestions (each checkable item is just a subquestion with value truthy, falsey, or longer detailed explanation), it wasn't worth creating a whole new subclass. We just look for the `checklist_group` question type here.